client/post-editor/controller.js

```diff
@@ -210,6 +210,7 @@ export default {
 		const postToCopyId = context.query.copy;
 
 		function startEditing( siteId ) {
+			const site = getSelectedSite( context.store.getState() );
```

Comment:
If the `siteId` is explicitly received as parameter, wouldn't it make more sense to call `getSite( state, siteId)` here? I know that @Copons gave you exactly the opposite advice (:smile:) and that in practice it doesn't really matter (the site will be always the selected one anyway). But it looks quite confusing to have a `siteId` parameter and then ignore it.